### DISCUSSION TITLE
Showing Users Table

### DISCUSSION THREAD

**Discussion by idofarit** [supabase/supabase#19911] [Questions] [answered]
How do I show Authenticated Users table into my app.

**Comment by GaryAustin1** [comment, marked as answer]
Normally you can’t use the auth schema thru the APIs.  You can use a View possibly or an rpc call to security definer function that returns the data.    The best way is to create a public profile table with a trigger on auth.users table. This https://supabase.com/docs/guides/auth/managing-user-data shows that method.

**Reply by GaryAustin1** [reply]
Not sure what you are saying by couldn't find the users table.  The trigger should be running on auth.users.

**Reply by idofarit** [reply]
Hi, Thanks for assissting. That worked fine..
Could you assist me how to perform SEARCH functionality from a table?

**Reply by GaryAustin1** [reply]
There are many ways to search a table from the client APIs or with rpc functions.  Not clear what you are trying to search, but should all use basic filters.
